assets/css/custom/notifications.scss

```diff
@@ -16,6 +17,54 @@
   color: white;
 }
 
+.notifications-header {
```

Comment:
lease remove all the old (now unused) css from this file (and other if necessary)